src/Server/ServerHelpers.ts

```diff
@@ -236,6 +237,15 @@ export function isBackdoorInstalled(server: BaseServer): boolean {
   return false;
 }
 
+export function isBackdoorInstalledInCompanyServer(companyName: string): boolean {
```

Comment:
Unless you have an immediate compelling use elsewhere, I'd keep this alongside the only thing that's using it (in `Company/utils.ts`), to avoid adding to our cross-helpers dependency web. In fact, I'd probably just merge the two functions together.

Reply:
Nevermind, since this uses `isBackdoorInstalled` there's no cutting this particular knot. I must have been extra sleepy last night.